ios/RNCallKeep/RNCallKeep.m

```diff
@@ -391,7 +391,7 @@ + (CXProviderConfiguration *)getProviderConfiguration:(NSDictionary*)settings
     providerConfiguration.maximumCallsPerCallGroup = 1;
     providerConfiguration.supportedHandleTypes = [NSSet setWithObjects:[NSNumber numberWithInteger:CXHandleTypePhoneNumber], nil];
     if (settings[@"supportsVideo"]) {
-        providerConfiguration.supportsVideo = settings[@"supportsVideo"];
+        providerConfiguration.supportsVideo = [settings[@"supportsVideo"] boolValue];
```

Comment:
Probably want to add a comment here about what this was doing, since it seems unrelated to this change. It's ok to do that on the actual PR we send to Wazo, but don't just ignore it.

Reply:
Ok I'll add some bullet points when we make the PR? Or do you mean comment in code.

Reply:
Yeah, commenting on the PR summary about what you're addressing is fine.